src/read/pe/export.rs

```diff
@@ -1,33 +1,36 @@
 use alloc::vec::Vec;
-use core::convert::TryInto;
 use core::fmt::Debug;
 
-use crate::read::ReadError;
-use crate::read::Result;
-use crate::ByteString;
-use crate::{pe, Bytes, LittleEndian as LE, U16Bytes, U32Bytes};
+use crate::read::{ByteString, Bytes, Error, ReadError, Result};
+use crate::{pe, LittleEndian as LE, U16Bytes, U32Bytes};
 
-/// Where an export is pointing to
-#[derive(Clone)]
+/// Where an export is pointing to.
+#[derive(Clone, Copy)]
 pub enum ExportTarget<'data> {
-    /// The export points at a RVA in the file
-    Local(u64),
-    /// The export is "forwarded" to another DLL
+    /// The address of the export, relative to the image base.
+    Address(u32),
+    /// Forwarded to an export ordinal in another DLL.
     ///
-    /// for example, "MYDLL.expfunc" or "MYDLL.#27"
-    Forwarded(&'data [u8]),
+    /// This gives the name of the DLL, and the ordinal.
+    ForwardByOrdinal(&'data [u8], u32),
+    /// Forwarded to an export name in another DLL.
+    ///
+    /// This gives the name of the DLL, and the export name.
+    ForwardByName(&'data [u8], &'data [u8]),
```

Comment:
Now this enum has three variants, maybe there could be a `.is_forwarded(&self)` that basically tells whether this is either `ForwardByName` or `ForwardByOrdinal`.